### ISSUE TITLE
[Bug] Mirroring ghosts on the mirror line

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#1686]
### Short Description of the issue:
Looks like the same issue as #1624 and #1566, however this only applies to bricks places within 0.5m of the mirror line. It doesn't happen 100% of the time though
https://i.imgur.com/LMZHS8J.mp4
### Steps to reproduce the issue:
Place and replace bricks within 0.5m of the mirror plane as shown in the clip

### Environment
- Skywanderers version: 20w14a
- Graphics Card: RX 580

**Comment by tsunamayo** [comment]
Thanks a lot for the video - very usefull.

**Comment by tsunamayo** [comment]
Okay I cant reproduce that one... So if you try with an axis a bit to the side. And also if you could send me your bp so I can check if I reproduce on your build.
Thanks!

**Comment by ProPeach** [comment]
Loaded up a new session and tried with the axis in the middle as it was before, to one side and in different orientations but I can't reproduce it. This makes me wonder if it only appears in longer sessions. I've included the two blueprints I was working on when I logged the issue, but they both showed this bug as have other ships in the past so I think it's universal.

[Skywanderers mirror lag blueprints.zip](https://github.com/tsunamayo/Skywanderers/files/4431148/Skywanderers.mirror.lag.blueprints.zip)
PS - Enter the yoke on the shuttle and hit the landing gear hotkey 😝
